contracts/interstake-yield-generator/src/msg.rs

```diff
@@ -50,6 +50,8 @@ pub enum ExecuteMsg {
         amount: Uint128,
         commission_address: Option<String>,
     },
+    /// Start unbonding current batch
+    Reconcile {},
```

Comment:
Look here at mister big brain eloquent. :D
But to be honest, I'm not sure this is the best name. So you changed `undelegate` to `query_undelegate`, and now undelegating is happening in `reconcile`?

Reply:
I got it from Larry, Blame it on him

Reply:
Btw i didnt change it to `query_undelegate` but to `queue_undelegate`